string[] link_names
float64[] joint_positions
geometry_msgs/Pose target_pose
---
moveit_msgs/RobotTrajectory[] trajectories
